### PR TITLE
STUD-166: Add tooltip to song error states in library

### PR DESCRIPTION
**Pull request by escobarjonatan**
Declined:
image
Error:
image

### CODE REVIEW

File: apps/studio/src/pages/home/library/MintingStatus.tsx [hunk 1]
```diff
@@ -34,6 +34,17 @@ const UI_MINTING_STATUS: Record<MintingStatusType, string> = {
...
+// Indicates all "An error occurred" statuses. Used externally to determine tooltip content.
+export const ErrorOccurredMintingStatuses = [
+  MintingStatusType.ArweaveUploadException,
+  MintingStatusType.DistributionException,
+  MintingStatusType.MintingException,
+  MintingStatusType.MintingPaymentException,
+  MintingStatusType.MintingPaymentTimeout,
+  MintingStatusType.ReleaseCheckException,
+  MintingStatusType.SubmittedForDistributionException,
+];
```
Comment (escobarjonatan) [inline apps/studio/src/pages/home/library/MintingStatus.tsx:46]: Added this here in order to keep minting ui status in one place so when we update the `UI_MINTING_STATUS` we can update this if needed.

**Review by scandycuz — APPROVED** [review]

File: apps/studio/src/pages/home/library/SongList.tsx [hunk 2]
```diff
@@ -31,7 +32,8 @@ import {
   SortOrder,
 } from "@newm-web/types";
 import { useNavigate } from "react-router-dom";
-import { MintingStatus } from "./MintingStatus";
+import { min } from "moment";
```
Comment (dmkirshon) [inline apps/studio/src/pages/home/library/SongList.tsx]: This should probably be removed, don't see the reference being used.

**Review by dmkirshon — CHANGES_REQUESTED** [review]
LGTM, just need to remove superfluous import.

File: apps/studio/src/pages/home/library/SongList.tsx (hunk 2 above)
Comment (escobarjonatan) [reply apps/studio/src/pages/home/library/SongList.tsx]: gotta love auto import as you type 🫠

**Review by dmkirshon — APPROVED** [review]
LGTM

**Pull request merged**